core/src/mindustry/ui/dialogs/JoinDialog.java

```diff
@@ -22,15 +22,15 @@
 import static mindustry.Vars.*;
 
 public class JoinDialog extends BaseDialog{
-    Seq<Server> servers = new Seq<>();
-    Dialog add;
-    Server renaming;
-    Table local = new Table();
-    Table remote = new Table();
-    Table global = new Table();
-    Table hosts = new Table();
-    int totalHosts;
-    int refreshes;
+    public Seq<Server> servers = new Seq<>();
+    public Dialog add;
+    public Server renaming;
+    public Table local = new Table(),
+        remote = new Table(),
+        global = new Table(),
+        hosts = new Table();
```

Comment:
I think it's better to leave old declaration

```suggestion
    public Table local = new Table();
    public Table remote = new Table();
    public Table global = new Table();
    public Table hosts = new Table();
```